php/types/events/register.php

```diff
@@ -113,12 +57,20 @@ function event_date_meta($event) {
                 add_meta_box('event_name', 'Event Name', 'event_name_meta', 'event', 'side', 'low');
                 add_meta_box('location', 'Event Location', 'event_location_meta', 'event', 'side', 'low');
                 add_meta_box('event_date', 'Event Date', 'event_date_meta', 'event', 'side', 'low');
-                add_meta_box('event_name', 'Event Name', 'event_name_meta', 'event', 'side', 'low');
-                add_meta_box('event_name', 'Event Name', 'event_name_meta', 'event', 'side', 'low');
-                add_meta_box('event_name', 'Event Name', 'event_name_meta', 'event', 'side', 'low');
-                add_meta_box('event_name', 'Event Name', 'event_name_meta', 'event', 'side', 'low');
-                add_meta_box('event_name', 'Event Name', 'event_name_meta', 'event', 'side', 'low');
 
              }
         )
-    );
\ No newline at end of file
+    );
+
+  register_meta('event', 'event_name', array(
+      'show_in_rest' => true
+  ));
+  register_meta('event', 'location', array(
+      'show_in_rest' => true
+  ));
+  register_meta('event', 'event_date', array(
+      'show_in_rest' => true
+  ));
+  register_meta('event', 'event_time', array(
```

Comment:
@josephpalma, can you finish meta registration for each rest field?

Reply:
Do you reference a schema somewhere? If you point me to the fields that need registration I can do this

Reply:
I've added documentation in #38.

Reply:
Looking at this again, sorry for the delay.

We're not using the Events type in Wordpress to create events, we use the KTUH google calendar, is this schema tied to that at all?

We are also not using the Latest Episodes type.

If we dont need these two types I would like to remove them. What do you think @dchan3 ?

Reply:
@dchan3 Idk why we are registering so many rest fields, when we only use the 4 mentioned here to sync with Google Calendar. In fact we're not using the Event type at all in the Wordpress dashboard. Can we remove the unused types? (including latest_epi)